src/scripts/main.js

```diff
@@ -1,28 +1,40 @@
 'use strict';
 
 const food = {
-  'Drink': {
-    'Wine': {},
-    'Schnaps': {},
+  Drink: {
+    Wine: {},
+    Schnaps: {},
   },
 
-  'Fruit': {
-    'Red': {
-      'Cherry': {},
-      'Strawberry': {},
+  Fruit: {
+    Red: {
+      Cherry: {},
+      Strawberry: {},
     },
-    'Yellow': {
-      'Banana': {},
-      'Pineapple': {},
+    Yellow: {
+      Banana: {},
+      Pineapple: {},
     },
```

Comment:
Don't name properties with capital letter. Just capitalize it where you need it.